src/puppetlabs/trapperkeeper/services/status/status_debug_logging.clj

```diff
@@ -0,0 +1,14 @@
+(ns puppetlabs.trapperkeeper.services.status.status-debug-logging
+  (:require [clojure.tools.logging :as log]
+            [schema.utils :refer [validation-error-explain]]
+            [cheshire.core :as json]
+            [puppetlabs.trapperkeeper.services.status.status-core :as status-core]))
+(defn log-status
+  "Log status information at the debug level as json
+
+  Note: This function is in its own namespace so that logback can use the namespace
+  as a way to route these log messages separately from other logging the application
+  might be doing, and so it shouldn't be moved from this namespace"
+  []
+  (let [status (status-core/status-latest-version :debug)]
```

Comment:
@cprice404, @jpinsonault - might be nice to have the ability to use TK-level parameters when doing the status logging, e.g., to be able to select a higher or lower level of granularity when logging rather than just hard-coding it to "debug".  Suppose we could just add other namespaces like this one later if we find the need, e.g., a "status-trace-logging" (assuming the 'trace' level is a thing, can't remember).  Figured I'd just throw that out for thoughts but I'm happy going with what we have already here if ye are.